src/main/resources/transformation/metafix-wip.fix

```diff
@@ -0,0 +1,27 @@
+/*
+  <entity name="TitleOther">
+    <entity name="title[]" flushWith="record">
+      <data name="" source="246?[ 345678].a">
+        <replace pattern="\.$" with="" />
+      </data>
+    </entity>
+  </entity>
+	*/
+
+# TODO: Das schreiben in einen tieferliegendes Array funktioniert anscheinend nicht.
+
+# set_array("TitleOther.title[]")
+# do list(path: "246?[ 345678]", "var": "varTitleOther")
```

Comment:
Could you name the `var` with a `$` sigil and single letter (usually `$i`)?

Reply:
Rationale: "local" variables (i.e., `bind` variables) have a short name that starts with a `$`, "global" variables (i.e., auxiliary record fields) have a descriptive name that starts with an `@`.

Reply:
I do